worker/evCheckerWorker/evCheckerWorker.go

```diff
@@ -0,0 +1,100 @@
+package evCheckerWorker
+
+import (
+	"bytes"
+	"encoding/pem"
+	"fmt"
+	"github.com/mozilla/tls-observatory/logger"
+	"github.com/mozilla/tls-observatory/worker"
+	"io/ioutil"
+	"os/exec"
+	"reflect"
+)
+
+var workerName = "evCheckerWorker"
+var workerDesc = `Provides insights for EV certificates`
+
+var log = logger.GetLogger()
+
+func init() {
+	worker.RegisterWorker(workerName, worker.Info{Runner: new(evWorker), Description: workerDesc})
```

Comment:
I'm not sure how to cleanly pass options to workers, considering they're on a different package and we initialize workers with `init()`. We could have it as an exported variable on the worker, but then `init()` will always execute before we get a chance to change the variable, and we can't really have the worker read it from main because then we'd have an import loop.